.github/CODEOWNERS

```diff
@@ -0,0 +1,38 @@
+# This file defines code owners for the OpenVINO GenAI repository
+
+# Default owners
+* @Wovchena @as-suvorov
+
+# CI
+/.github/ @akashchi
+
+# JavaScript
+/src/js/ @Retribution98 @yatarkan
+/samples/js/ @Retribution98 @yatarkan
+
+# Documentation
+/site/ @yatarkan
+
+/src/cpp/include/openvino/genai/image_generation/ @likholat
+/src/cpp/include/openvino/genai/rag/ @as-suvorov
+# /src/cpp/include/openvino/genai/speculative_decoding/
+/src/cpp/include/openvino/genai/speech_generation/ @rkazants @as-suvorov
+/src/cpp/include/openvino/genai/visual_language/ @yatarkan
+
+# /src/cpp/src/continuous_batching/
+# /src/cpp/src/gguf_utils/
+/src/cpp/src/image_generation/ @likholat
+# /src/cpp/src/llm/
+/src/cpp/src/lora/ @likholat
+/src/cpp/src/prompt_lookup/ @as-suvorov
+/src/cpp/src/rag/ @as-suvorov
+/src/cpp/src/sampling/structured_output/ @apaniukov @pavel-esir
+# /src/cpp/src/sampling
```

Comment:
Add at leas someone to commented out components so it's not just me and you. I think the same is for prompt_lookup but rag stays on you